src/js/schemas/service-schema/Networking.js

```diff
@@ -83,11 +83,9 @@ const Networking = {
         });
       },
       filterProperties(service = {}, instanceDefinition, model) {
-        let properties = Networking
-          .properties
-          .ports
-          .itemShape
-          .properties;
+        let {properties} = Networking.properties.ports.itemShape;
```

Comment:
All in this file is basically just making it easier to understand what is going on